server/odc-core/src/main/resources/i18n/ErrorMessages.properties

```diff
@@ -16,7 +16,7 @@ com.oceanbase.odc.ErrorCodes.UnauthorizedSessionAccess=Unauthorized session acce
 com.oceanbase.odc.ErrorCodes.PermissionChanged=Permission has been changed, please exit and try again
 com.oceanbase.odc.ErrorCodes.UnauthorizedDataAccess=Unauthorized data access
 com.oceanbase.odc.ErrorCodes.OverLimit=Cannot process due {0} exceed limit, maximum was {1}
-com.oceanbase.odc.ErrorCodes.AttemptLoginOverLimit=Cannot process due {0} exceed limit, maximum was {1}, please try {2} second(s) later
+com.oceanbase.odc.ErrorCodes.AttemptLoginOverLimit=Cannot process due {0} exceed limit, maximum was {1}, please try {2} second(s) later, or contact the administrator to modify the system configuration
```

Comment:
here we may not change error message, or we are encouraging user to disturb the administrator

Reply:
done